frontend/src/Layout/StudentDashBoard/StudentDashBoard.jsx

```
@@ -0,0 +1,239 @@
+        { type: "link", name: "Settings", path: "/settings" },
+        // { type: "link", name: "Logout", path: "/logout", specialClass: "bg-red-600 hover:bg-red-700" },
+    ];
+
+    return (
+        <div className="">
+            {/* Navbar (Fixed at top) */}
+            <nav className="w-full bg-indigo-800 fixed top-0 z-20 text-white flex justify-between items-center shadow-md">
+                <button
+                    className="p-2 bg-indigo-600 text-white rounded-md shadow-md md:hidden"
+                    onClick={() => setIsSidebarOpen(!isSidebarOpen)}
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)